lib/cargo.js

```diff
@@ -23,12 +30,19 @@ function provideBuilder() {
 
     settings: function (path) {
       var cargoPath = atom.config.get('build-cargo.cargoPath');
+      var with_verbose = function (name) {
+        if (atom.config.get('build-cargo.verbose')) {
+          return [name, '--verbose'];
+        } else {
+          return [name];
+        }
+      };
 
```

Comment:
A better way to structure this would be to do:
```
let args = [];
atom.config.get('build-cargo.verbose') && args.push('--verbose');
```
Then when creating the JSON structs, set `args: [ 'build' ].concat(args)`

This way, it's easy to add other possible configurations which modify args.